tests/src/performance/linkbench/setup.cmd

```diff
@@ -0,0 +1,109 @@
+@echo off
+setlocal
+call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\Tools\VsDevCmd.bat"
+
+set ROOT=%cd%\LinkBench
+set ExitCode=0
+mkdir LinkBench 2> nul
+pushd %ROOT%
+
+echo Setting up  ** HelloWorld **
+cd %ROOT%
+git clone http://github.com/swaroop-sridhar/hello.git HelloWorld -b dev
+cd HelloWorld
+dotnet restore -r win10-x64
+dotnet publish -c release -r win10-x64
+dotnet msbuild /t:Link /p:LinkerMode=sdk /p:RuntimeIdentifier=win10-x64 /v:n /p:Configuration=release
+if errorlevel 1 set ExitCode=1 
+echo -- Done -- 
+
+echo Setting up  ** MusicStore **
+cd %ROOT%
+git clone https://github.com/swaroop-sridhar/JitBench -b dev
+cd JitBench\src\MusicStore
+dotnet restore -r win10-x64 
+dotnet publish -c release -r win10-x64
+dotnet msbuild /t:Link /p:LinkerMode=sdk /p:RuntimeIdentifier=win10-x64 /v:n /p:LinkerRootFiles=MusicStoreReflection.xml /p:Configuration=release
+if errorlevel 1 set ExitCode=1 
+echo -- Done -- 
+
+echo Setting up  ** MusicStore Ready2Run **
+cd %ROOT%\JitBench\src\MusicStore
+powershell -noprofile -executionPolicy RemoteSigned -file Get-Crossgen.ps1
+pushd  bin\release\netcoreapp2.0\win10-x64\
+call :SetupR2R publish_r2r
+if errorlevel 1 set ExitCode=1 
+call :SetupR2R linked_r2r
+if errorlevel 1 set ExitCode=1 
+echo -- Done -- 
+
+echo Setting up ** CoreFX **
+cd %ROOT%
+git clone http://github.com/dotnet/corefx
+cd corefx
+set BinPlaceILLinkTrimAssembly=true
+call build.cmd -release
+if errorlevel 1 set ExitCode=1 
+echo -- Done -- 
+
+echo ** Roslyn **
+cd %ROOT%
+git clone https://github.com/swaroop-sridhar/roslyn.git -b dev
```

Comment:
The dev branch in my fork has code to fetch the linker from the package and the roslyn roots etc.
https://github.com/dotnet/roslyn/compare/master...swaroop-sridhar:dev